pkg/istio-agent/agent.go

```diff
@@ -53,7 +52,7 @@ import (
 //
 // Or disable the jwt validation while debugging SDS problems.
 
-var (
+const (
```

Comment:
Why ? We may still allow code ( like main ) to override those.

Reply:
I think if these are modified they should be passed through security.Options rather than modifying global state, or it becomes harder to test/understand